contrib/playlib/src/mill/playlib/Version.scala

```diff
@@ -0,0 +1,17 @@
+package mill
+package playlib
+
+import mill.define.Target
+import mill.scalalib._
+
+private[playlib] trait Version extends Module{
+  def playVersion: T[String]
+
+  private[playlib] def playMinorVersion: T[String] = T {
```

Comment:
The name suggests, it refers to the single minor version part. Is it possibly a `playBinVersion`? Why do we add the trailing `.0` ?

Reply:
I think minor is correct here, play uses semantic versioning which defines versions as: 
`<major>.<minor>.<patch>` 
This method returns the minor version. I initially added the `.0` because I wasn't sure what I was doing with all the worker key computation for the cross build and bridge worker loading. 
I now realise that the `.0` is useless and can be dropped.